blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpIntegrationTestBase.java

```diff
@@ -17,44 +17,55 @@
 package com.rackspacecloud.blueflood.http;
 
 import com.github.tlrx.elasticsearch.test.EsSetup;
+import com.rackspacecloud.blueflood.inputs.formats.JSONMetricsContainerTest;
 import com.rackspacecloud.blueflood.inputs.handlers.HttpEventsIngestionHandler;
 import com.rackspacecloud.blueflood.inputs.handlers.HttpMetricsIngestionServer;
 import com.rackspacecloud.blueflood.io.*;
 import com.rackspacecloud.blueflood.service.*;
+import com.rackspacecloud.blueflood.types.Event;
 import com.rackspacecloud.blueflood.utils.ModuleLoader;
+import org.apache.commons.io.IOUtils;
 import org.apache.http.HttpEntity;
 import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.client.utils.URIBuilder;
 import org.apache.http.entity.ContentType;
 import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.DefaultHttpClient;
 import org.junit.*;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.io.*;
 import java.net.URISyntaxException;
 import java.util.Collection;
 import java.util.HashSet;
+import java.util.Random;
 
+import static com.rackspacecloud.blueflood.inputs.handlers.wrappers.TestGsonParsing.getJsonFromFile;
 import static org.mockito.Mockito.spy;
 
 public class HttpIntegrationTestBase {
+
+    public static final long TIME_DIFF = 2000;
```

Comment:
Are these milli seconds? Can you change TIME_DIFF to TIME_DIFF_IN_MILLIS or write a comment?